transmission_interface/include/transmission_interface/four_bar_linkage_transmission.hpp

```diff
@@ -289,7 +289,7 @@ inline void FourBarLinkageTransmission::actuator_to_joint()
 
     joint_eff[0].set_value(jr[0] * act_eff[0].get_value() * ar[0]);
     joint_eff[1].set_value(
-      jr[1] * (act_eff[1].get_value() * ar[1] - act_eff[0].get_value() * ar[0] * jr[0]));
+      jr[1] * (act_eff[1].get_value() * ar[1] - jr[0] * act_eff[0].get_value() * ar[0]));
```

Comment:
Was this change intentional?

Reply:
Yes. It seems to reduce the rounding error. Notice how jr[0], act_eff[0], a[0] are now in the same order as they are on line 290.

Reply:
ok, I see.